src/listener_builder.rs

```diff
@@ -163,6 +163,14 @@ macro_rules! make_listener_builder {
                 builder.forwards_to(forwards_to);
                 self
             }
+            #[napi]
+            pub fn policy(&mut self, policy: String) -> Result<&Self> {
```

Comment:
This looks good for the builder pattern config, but there is also the `connect` side for one-liners, so policy should likely be added to the connect common as well. There's macro magic for calling the regular configuration options, but this returns a result, so I think quickest path is likely to just add the function directly in that config_common:

https://github.com/ngrok/ngrok-javascript/blob/1f9a6f9b08519df5b2aef3947e64a4bb8b5c760d/src/connect.rs#L87

Reply:
`ngrok-python` needed the same thing, here's an example of that change there:
[connect.rs](https://github.com/ngrok/ngrok-python/pull/92/files#diff-4e7bfdc1a4370e34beb83e080ba29965735d732b714e5dde381bee13daec97b9R119)
[test-connect](https://github.com/ngrok/ngrok-python/pull/92/files#diff-ff9bd5d923dd685930883c21dd5361a0d9a124a3920a94b79a7490ee8a403414R180)

Reply:
Good catch, added!